vitest.config.ts

```
@@ -0,0 +1,20 @@
+import { defineConfig } from 'vitest/config'
```

ESLint

[error] Strings must use doublequote.

(@typescript-eslint/quotes)